src/components/BottomSheetInLineNotification.tsx

```diff
@@ -54,35 +57,43 @@ const BottomSheetInLineNotification = ({
     }
   )
 
+  const AnimatedNotification = (
+    <Animated.View style={[styles.notificationContainer, animatedStyle]}>
+      <InLineNotification {...inLineNotificationProps} />
+    </Animated.View>
+  )
+
   if (!isVisible) {
     return null
   }
 
-  return (
-    <SafeAreaView edges={['bottom']} style={[styles.modal, styles.container]}>
-      <Animated.View style={[styles.modal, styles.background, animatedOpacity]} />
-      <Animated.View style={[styles.notificationContainer, animatedStyle]}>
-        <InLineNotification {...inLineNotificationProps} />
-      </Animated.View>
-    </SafeAreaView>
-  )
+  if (withBackdrop) {
+    return (
+      <SafeAreaView edges={['bottom']} style={[styles.modal, styles.container]}>
+        <Animated.View style={[styles.modal, styles.background, animatedOpacity]} />
+        {AnimatedNotification}
+      </SafeAreaView>
+    )
+  }
+
+  return AnimatedNotification
 }
 
 const styles = StyleSheet.create({
   modal: {
     ...StyleSheet.absoluteFillObject,
   },
   container: {
-    paddingHorizontal: Spacing.Regular16,
     alignItems: 'center',
   },
   background: {
     backgroundColor: Colors.black,
   },
   notificationContainer: {
     position: 'absolute',
-    bottom: Spacing.Large32,
-    width: '100%',
+    bottom: Spacing.XLarge48,
+    left: Spacing.Regular16,
+    width: variables.width - Spacing.Large32,
```

Comment:
nit: would this work?
```suggestion
    left: Spacing.Regular16,
    right: Spacing.Regular16,
```

Reply:
i think so, but i ended up using the new Toast component after it was merged so didn't need changes to this file after all :)